src/sentry/integrations/opsgenie/integration.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import annotations
+
+# import logging
+from typing import Any, Mapping, Sequence
+
+# from django import forms
+from django.utils.translation import gettext_lazy as _
+
+# from sentry import options
+from sentry.integrations.base import (
+    FeatureDescription,
+    IntegrationFeatures,
+    IntegrationInstallation,
+    IntegrationMetadata,
+    IntegrationProvider,
+)
+
+# from sentry.models import Integration, OrganizationIntegration
+from sentry.pipeline import PipelineView
+
+DESCRIPTION = """
+Trigger alerts in Opsgenie from Sentry.
+
+OpsGenie is a cloud-based service for dev & ops teams, providing reliable alerts, on-call schedule management and escalations.
+OpsGenie integrates with monitoring tools & services and ensures that the right people are notified via email, SMS, phone calls,
+and iOS & Android push notifications.
+"""
+
+
+FEATURES = [
+    FeatureDescription(
+        """
+        Manage incidents and outages by sending Sentry notifications to Opsgenie.
+        """,
+        IntegrationFeatures.INCIDENT_MANAGEMENT,
+    ),
+    FeatureDescription(
+        """
+        Configure Sentry rules to trigger notifications based on conditions you set.
+        """,
+        IntegrationFeatures.ALERT_RULE,
+    ),
+]
+
+# need to add feature flag?
+metadata = IntegrationMetadata(
+    description=DESCRIPTION.strip(),
+    features=FEATURES,
+    author="The Sentry Team",
+    noun=_("Installation"),
+    issue_url="https://github.com/getsentry/sentry/issues/new?assignees=&labels=Component:%20Integrations&template=bug.yml&title=Integration%20Problem",
+    source_url="https://github.com/getsentry/sentry/tree/master/src/sentry/integrations/opsgenie",  # currently incorrect, as it hasn't been merged
+    aspects={},
+)
+
+
+class OpsgenieIntegration(IntegrationInstallation):
+    pass
+
+
+class OpsgenieIntegrationProvider(IntegrationProvider):
+    key = "opsgenie"
+    name = "Opsgenie (Integration)"
+    metadata = metadata
+    integration_cls = OpsgenieIntegration
+    features = frozenset([IntegrationFeatures.INCIDENT_MANAGEMENT, IntegrationFeatures.ALERT_RULE])
+    # requires_feature_flag = True  # limited release
+
+    def get_pipeline_views(self) -> Sequence[PipelineView]:
```

Comment:
Do we need to override the parent method if we're not doing anything differently?